src/serviceWorker.tsx

```diff
@@ -76,10 +74,11 @@ function registerValidSW(swUrl: string, config?: Config) {
               // At this point, the updated precached content has been fetched,
               // but the previous service worker will still serve the older
               // content until all client tabs are closed.
-              push(
-                "New content is available and will be used when all " +
-                  "tabs for this page are closed.",
-              );
+
+              // console.log(
+              //   "New content is available and will be used when all " +
+              //     "tabs for this page are closed.",
+              // );
```

Comment:
the comment above is already enough...! Why having commented out `console.log`?! Either just the text or nothing..